### DISCUSSION TITLE
Change Landing Page

### DISCUSSION THREAD

**Discussion by jokerigno** [Athou/commafeed#1185] [General]
Hi, I just installed a dockerized version of Commafeed and the first thing I see is the welcome page with the login button at the top,

Can the /#/login become the main landing page somehow?

**Comment by Athou** [comment]
It's not possible at the moment, but I think it's a good idea to redirect to the login page if `allowRegistrations` is set to `false`, which is the the default value. I'll see what I can do.

**Reply by Athou** [reply]
Done in https://github.com/Athou/commafeed/commit/8297edaf71cf8a468e9f53f82d5ccfe7f1969fdb it will be part of v4.1.0
